README.md

```diff
@@ -401,3 +403,19 @@ tilesets tilejson <tileset_id>
 Flags:
 
 * `--secure`: By default, resource URLs in the retrieved TileJSON (such as in the "tiles" array) will use the HTTP scheme. Include this query parameter in your request to receive HTTPS resource URLs instead.
+
+### list-activity
+
+List tileset activity for an account. Returns a pagination key `next` if there are more results than the return limit that can be passed into another command as the `start` argument.
```

Comment:
Hi there - we know what activity means, but could we clarify it for users? i.e. total requests for a tileset over the past 30 days.